bin/raml2html

```diff
@@ -74,6 +83,34 @@ function writeOutput(result, config, argv) {
   });
 }
 
+function errorPath(error) {
```

Comment:
I think we have an issue here.
We should use path from error.trace, instead of error iself, because now error is point to parent type, instead of real place of error.
I will look at this

Reply:
I already merged to develop, but I'll wait with merging to master and releasing this. Let me know what you find!

Reply:
ok